.github/workflows/build-test.yaml

```diff
@@ -8,14 +8,31 @@ name: test
 
 jobs:
   build_and_test:
-    name: Rust project
-    runs-on: ubuntu-latest
+    strategy:
```

Comment:
can you enable prs into mrc-4553 to build please, that will test the python build

Reply:
It was only targeting mrc-4553 while I was waiting for that to build. I've rebased it onto master now.